## 取り組んだ課題一覧 
- Python インプット
## わかったこと
- 例外処理のexceptでは例外を指定しないと全ての例外をキャッチして適切にエラーハンドリングできないため`exept <例外> as e`の形で記述する
- finallyの処理はtry except elseの中で例外が起こるとfinallyの中身を実行してから例外を発生させる
- tryでbreakやcontinue,returnに達してもfinallyが実行される
- raiseをtryで発生させることで簡単にexceptのテストを行うことができる
- tracebackモジュールを使用することでエラーの内容を確認することができる
- fileはopen関数で内容を取得することができる。close()をセットで行う必要があるがwithステートメントを使用すると処理が終わると自動でcloseしてくれる
- csvモジュールはlistではなくdictionaryのメソッドで扱うようにする
## 次やること  
- Python インプット
## 感じたこと 
- 3/4までインプットを進めることができた。あと少しなのでもう一踏ん張り頑張りたい。